stylesheets/_component.rules.scss

```diff
@@ -298,3 +298,306 @@ td.table__form .rules + .form__group:last-of-type {
         width: auto;
     }
 }
+
+/**
+ * The following version 2 was developed to meet the needs of XFP conditional logic rules
+ */
+.rules-container--v2 {
+    border-left: 4px solid color(gray, lightest);
+    padding-left: 20px;
+    position: relative;
+
+    &::before,
+    &::after {
+        background-color: color(white);
+        content: '';
+        display: block;
+        height: 35px;
+        left: -4px;
+        position: absolute;
+        width: 24px;
+    }
+
+    &::before {
+        border-bottom: 4px solid color(gray, lightest);
+        top: 0;
+    }
+
+    &::after {
+        border-top: 4px solid color(gray, lightest);
+        bottom: 0;
+    }
+
+    .rule-links button {
```

Comment:
I'd BEM the button - `.rule-links__button`.

Reply:
Because this is an in-flight project I'm going to use `.rule-link .btn` instead, which means I don't have to trigger a round of re-tests for the XFP team.